src/main/drivers/bus_spi_hal.c

```diff
@@ -260,27 +261,21 @@ bool spiIsBusBusy(SPI_TypeDef *instance)
         return false;
 }
 
-bool spiTransfer(SPI_TypeDef *instance, uint8_t *out, const uint8_t *in, int len)
+bool spiTransfer(SPI_TypeDef *instance, const uint8_t *in, uint8_t *out, int len)
```

Comment:
`in` and `out` are good for semantics, like they are for reference types in C# and similar languages. e.g. `in const uint8_t* rxData, out uint8_t* txData`, given:
```
#define in
#define out
```
Do nothing, but it's a synthetic sugar and not everyone like it.